docs/source/host_challenge.md

```diff
@@ -110,6 +110,13 @@ EvalAI supports all kinds of HTML tags which means you can add images, videos, t
 
 Finally run the `./run.sh` script in the bundle. It will generate a `challenge_config.zip` file that contains all the details related to the challenge. Now, visit [EvalAI - Host challenge page](https://evalai.cloudcv.org/web/challenge-host-teams) and select/create a challenge host team. Then upload the `challenge_config.zip`.
 
+### Step 6: (Optional) Set host submission as baseline submission
```

Comment:
Hey, @Mukul2000 Thank you for the efforts. Actually, we don't want to add it here but it would be good to add it as a separate page with screenshots like: https://evalai.readthedocs.io/en/latest/approve_challenge.html

Reply:
Hi, thanks for taking the time to review it, I'll make the necessary changes.

Reply:
Hi, I have made the requested changes and made a seperate step by step page for the same. Please check.